packages/pipeline/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+    "extends": "../../tsconfig",
+    "compilerOptions": {
+        "outDir": "lib"
```

Comment:
How does this compile with js files without allowJS flag?
BTW, why can't we make that file above a ts file?

Reply:
We should convert it to TS.